__tests__/app.test.js

```diff
@@ -98,3 +88,34 @@ describe("GET: 200 /api/articles/:article_id", () => {
     });
   });
 });
+
+describe("GET: 200 /api/articles", () => {
+  test("Should return the articles array sorted in descending order and without a body property", () => {
+    return request(app)
+      .get("/api/articles")
+      .expect(200)
+      .then(({ body }) => {
+        const articles = body.articles;
+        expect(articles).toHaveLength(13);
+        const articlesSortedByDate = articles.sort((a, b) =>
+          a.created_at.localeCompare(b.created_at)
+        );
+        expect(articlesSortedByDate).toEqual(articles);
```

Comment:
This won't work because sort will also sort the original array so they will both be changed. Also `localecCompare` will sort them as strings which may not be the same as by date, look to use something like jestSorted.